modules/javafx.graphics/src/main/java/javafx/scene/image/WritableImage.java

```diff
@@ -145,17 +172,33 @@ boolean pixelsReadable() {
         return true;
     }
 
+    void bufferDirty(Rectangle rect) {
+        getWritablePlatformImage().bufferDirty(rect);
+        pixelsDirty();
+    }
+
+    private static PixelBuffer validatePixelBuffer(PixelBuffer pixelBuffer) {
+        Objects.requireNonNull(pixelBuffer, "PixelBuffer must be specified.");
+        return pixelBuffer;
+    }
```

Comment:
You can directly return:

    return Objects.requireNonNull(pixelBuffer, "PixelBuffer must be specified.");

In fact, this method might not be needed since it's used only once and does a single check. The constructor could be

```
public WritableImage(@NamedArg("PixelBuffer") PixelBuffer pixelBuffer) {
    super(Objects.requireNonNull(pixelBuffer, "PixelBuffer must be specified."));
```

though this can be stylistically unappealing.

Reply:
Changed as `return Objects.requireNonNull(pixelBuffer, "PixelBuffer must be specified.");`